src/Symfony/Bridge/Twig/Extension/WorkflowExtension.php

```diff
@@ -45,6 +46,14 @@ public function getEnabledTransitions($object, $name = null)
         return $this->workflowRegistry->get($object, $name)->getEnabledTransitions($object);
     }
 
+    public function hasPlace($object, $state, $name = null)
```

Comment:
BTW, it's weird to call the variable `state` when the twig function, php method call it `place`.
Could you update it please?